Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -314,23 +324,46 @@ void OpenALStream::SoundLoop()
     // many silence samples.  These do not need to be timestretched.
     if (rate > 0.10)
     {
-      soundTouch.setTempo(rate);
-      if (rate > 10)
+      if (use_timestretching)
+      {
+        soundTouch.setTempo(rate);
+      }
+      else
+      {
+        alSourcef(ui_source, AL_PITCH, static_cast<ALfloat>(rate));
```

Comment:
Oh right, urgh. So on macOS, at least, every backend sounds "buttery smooth," as you said. Not sure what's up with XAudio2.